testing/extended/webhook_direct_post.py

```
@@ -0,0 +1,173 @@
+    headers = {"Content-Type": "application/json"}
+    if not args.no_auth_header:
+        headers["Authorization"] = f"Bearer {bearer}"
+
+    url = f"{wp_url}/wp-json/swh/v1/inbound-email"
+    print(f"POST {url}")
+    print(f"  auth-header   : {'omitted' if args.no_auth_header else 'present'}")
+    print(f"  bearer        : {'real' if bearer == (args.bearer or '') else 'real (resolved)'}")
+    print(f"  payload keys  : {list(payload.keys())}")
+
+    resp = requests.post(url, headers=headers, json=payload, timeout=15)
+    print(f"\nResponse:")
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)